mmv1/third_party/terraform/tests/data_source_google_compute_region_network_endpoint_group_test.go.erb

```diff
@@ -0,0 +1,83 @@
+<% autogen_exception -%>
```

Comment:
This file does not use any ruby, you can make this a `.go` file instead of a `.go.erb`